e2e/cypress/integration/dataset.spec.js

```diff
@@ -0,0 +1,60 @@
+describe('Dataset', () => {
+
+    before(() => {
+        cy.login()
+        cy.delete_organization('test-organization')
```

Comment:
Just be aware that if test-organization has some other dataset in it, this will fail.  I know that all of the tests remove datasets after the test is complete.  But if there is some external interaction, the `delete_organization` will fail.  For example, if you have inventory app up with some dataset in it and then you try to run the cypress tests, it won't delete the organization and then when it tries to create the organization, it'll say it has a conflict.  Should we try to call the `reset_db` functionality instead?

Reply:
That is fair. To mitigate as much as possible, the `delete_organization` command _tries_ to delete the organization, but doesn't hard fail if it can't. There may be some unintended consequences, I'll make sure to document in the readme.